include/miral/miral/window_specification.h

```diff
@@ -56,7 +56,9 @@ class WindowSpecification
     auto operator=(WindowSpecification const& that) -> WindowSpecification&;
 
     WindowSpecification(mir::shell::SurfaceSpecification const& spec);
+    /// \deprecated and rendered inert in MirAL 3.3
     WindowSpecification(mir::scene::SurfaceCreationParameters const& params);
+    /// \deprecated and rendered inert in MirAL 3.3
     void update(mir::scene::SurfaceCreationParameters& params) const;
```

Comment:
These cannot be called by client code as it has no access to `mir::scene::SurfaceCreationParameters`, so while dropping them is technically an ABI break I don't think we should go to any effort to "deprecated and rendered inert".